config/routes.rb

```diff
@@ -29,6 +29,7 @@
 
   root 'external#index'
   get 'dashboard' => 'dashboard', as: :dashboard
+  get 'site_map' => 'external#site_map', as: :site_map
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.